code/modules/unit_tests/unit_test.dm

```diff
@@ -28,6 +28,8 @@ GLOBAL_VAR(test_log)
 	//internal shit
 	var/focus = FALSE
 	var/succeeded = TRUE
+	//Do we have our own handling?
+	var/snowflake = FALSE
```

Comment:
This is a dumb variable name, name it what it actually is. Just do `auto_run = TRUE`.

Reply:
it seems like you want a priority system here instead, so let me nuke it totally